games-kids/tuxmath/tuxmath-1.7.2.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="Tux4Kids' math tutor for kids"
+DESCRIPTION="TuxMath is an educational math tutor for kids starring Tux, \
+the Linux penguin. This educational game comes with two different games for \
+practicing math, and having a great time doing it."
+HOMEPAGE="https://tux4kids.alioth.debian.org/"
+COPYRIGHT="2000-2009 Sam Hart"
+LICENSE="GNU GPL v2"
+REVISION="2"
+SOURCE_URI="https://sf.net/projects/tuxmath/files/tuxmath-source/TuxMath%201.7.2%20-%20Source/tuxmath_w_fonts-1.7.2.tar.gz"
+CHECKSUM_SHA256="7a3e39b17b88402a89780ac56368d8d92322c3b91f50ee4f9b747a8e2901b3a4"
+SOURCE_DIR="tuxmath_w_fonts-$portVersion"
+ADDITIONAL_FILES="tuxmath.rdef"
+PATCHES="tuxmath_x86-$portVersion.patchset"
```

Comment:
dropped the _x86 in the patchset, didn't know about the 'git mv' ... will try in a sec (hope it workes as should) :)